sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,513 @@
+-- Query for fenix_derived.feature_usage_metrics_v1
+-- For more information on writing queries see:
+-- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE start_date DATE DEFAULT "2021-05-05";
+
+DECLARE end_date DATE DEFAULT current_date;
+
+WITH dau_segments AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS dau
+  FROM
+    fenix.metrics
+  WHERE
+    DATE(submission_timestamp) >= start_date
+  GROUP BY
+    submission_date
+),
+product_features AS (
+  SELECT
+    client_info.client_id,
+    DATE(submission_timestamp) AS submission_date,
+    --Credential Management: Logins
+    COALESCE(SUM(metrics.counter.logins_deleted), 0) AS logins_deleted,
+    COALESCE(SUM(metrics.counter.logins_modified), 0) AS logins_modified,
+    COALESCE(SUM(metrics.quantity.logins_saved_all), 0) AS currently_stored_logins,
+    --Credential Management: Credit Cards
+    COALESCE(SUM(metrics.counter.credit_cards_deleted), 0) AS credit_cards_deleted,
+    COALESCE(SUM(metrics.quantity.credit_cards_saved_all), 0) AS currently_stored_credit_cards,
+    --Credential Management: Addresses
+    COALESCE(SUM(metrics.counter.addresses_deleted), 0) AS addresses_deleted,
+    COALESCE(SUM(metrics.counter.addresses_updated), 0) AS addresses_modified,
+    COALESCE(SUM(metrics.quantity.addresses_saved_all), 0) AS currently_stored_addresses,
+    --Bookmark
+    COALESCE(
+      SUM(metrics_bookmarks_add_table.value),
+      0
+    ) AS bookmarks_add,
+    COALESCE(
+      SUM(metrics_bookmarks_delete_table.value),
+      0
+    ) AS bookmarks_delete,
+    COALESCE(
+      SUM(metrics_bookmarks_edit_table.value),
+      0
+    ) AS bookmarks_edit,
+    COALESCE(
+      SUM(metrics_bookmarks_open_table.value),
+      0
+    ) AS bookmarks_open,
+    COALESCE(
+      SUM(metrics.counter.metrics_desktop_bookmarks_count),
+      0
+    ) AS metrics_desktop_bookmarks_count,
+    COALESCE(
+      SUM(metrics.counter.metrics_mobile_bookmarks_count),
+      0
+    ) AS metrics_mobile_bookmarks_count,
+    COALESCE(
+      SUM(CASE WHEN metrics.boolean.metrics_has_desktop_bookmarks IS TRUE THEN 1 ELSE 0 END),
+      0
+    ) AS metrics_has_desktop_bookmarks,
+    COALESCE(
+      SUM(CASE WHEN metrics.boolean.metrics_has_mobile_bookmarks IS TRUE THEN 1 ELSE 0 END),
+      0
+    ) AS metrics_has_mobile_bookmarks,
+    --Privacy
+    COALESCE(
+      SUM(
+        CASE
+          WHEN metrics.string.preferences_enhanced_tracking_protection = 'standard'
+            THEN 1
+          ELSE 0
+        END
+      ),
+      0
+    ) AS etp_standard,
+    COALESCE(
+      SUM(
+        CASE
+          WHEN metrics.string.preferences_enhanced_tracking_protection = 'strict'
+            THEN 1
+          ELSE 0
+        END
+      ),
+      0
+    ) AS etp_strict,
+    COALESCE(
```

Comment:
Is the COALESCE needed here? The case can return two possible values `1` and `0`, wouldn't we always end up with a number?